File: app/src/main/java/dev/ragnarok/fenrir/mvp/view/IPostCreateView.java
package dev.ragnarok.fenrir.mvp.view;

import android.net.Uri;

import androidx.annotation.NonNull;

import java.util.List;


public interface IPostCreateView extends IBasePostEditView, IToolbarView {
    void goBack();

    void displayUploadUriSizeDialog(@NonNull List<Uri> uris);
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/fragment/attachments/postcreate/IPostCreateView.kt
package dev.ragnarok.fenrir.fragment.attachments.postcreate

import android.net.Uri
import dev.ragnarok.fenrir.fragment.attachments.abspostedit.IBasePostEditView
import dev.ragnarok.fenrir.fragment.base.core.IToolbarView

interface IPostCreateView : IBasePostEditView, IToolbarView {
    fun goBack()
    fun displayUploadUriSizeDialog(uris: List<Uri>)
}